api/serverpb/server.proto

```diff
@@ -291,6 +291,61 @@ message AWSInstanceCheckRequest {
 
 message AWSInstanceCheckResponse {}
 
+message GetFileRequest {
+  string name = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+}
+
+message GetFileResponse {
+  // name represent file name with extension.
+  string name = 1;
+  // content represent file content.
+  bytes content = 2;
+}
+
+message InsertFileRequest {
+  string name = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  bytes content = 2 [
+    (validator.field) = {
+      length_gt: 0
+    }
+  ];
+}
+
+message UpdateFileRequest {
+  string old_name = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  string new_name = 2;
+  bytes content = 3;
+}
+
+message DeleteFileRequest {
+  string name = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+}
+
+message ListFilesResponse {
+  repeated string names = 1;
+}
```

Comment:
we don't need CRUD, because we have a hardcoded list of files, so we don't need so much endpoints. List and Update should be enough

Reply:
What should list return then? Names would be enough? Update should contain old name, new name if we hardcode file names?

Reply:
Which fields do you think would be useful in the future to update?

Reply:
for both endpoints, there might be just 2 fields `prometheus` and `alertmanager`

Reply:
I would like to discuss model will looks like then.

Reply:
after approvals in the slack, it was decided to leave only 2 endpoints and models for them